lib/tasks/hyacinth/setup.rake

```diff
@@ -56,5 +37,107 @@ namespace :hyacinth do
         end
       end
     end
+
+    desc "Add Descriptive Metadata category and Title dynamic fields"
+    task seed_dynamic_field_entries: :environment do
+      user = User.find_by(email: default_user_accounts[0][:email]);
+      if !user
+        puts Rainbow("seed_dynamic_field_entries requires default Admin user. Run default_users task first").red.bright
+        next
+      end
+      df_category = DynamicFieldCategory.find_by(display_label: "Descriptive Metadata")
+      if df_category
+        puts Rainbow("dynamic field category 'Descriptive Metadata' already exists (skipping).").blue.bright
+      else
+        df_category = DynamicFieldCategory.create!(display_label: "Descriptive Metadata")
+        puts Rainbow("Created dynamic field category 'Descriptive Metadata'").green
+      end
+
+      df_group = DynamicFieldGroup.find_by(display_label: "Title")
```

Comment:
For DynamicFieldGroup and DynamicField lookups should be done by `:string_key`. In this case, for those two models `string_keys` have to be unique.